### ISSUE TITLE
Autocomplete: scrollbar not usable

### ISSUE THREAD

**Issue by Olivier-kel** [quasarframework/quasar#3063]
Hi,

I have a small issue with the autocomplete component.
If the popover of the results has a scrollbar, using it closes the popover... before to have a chance to select an entry.

I'm using the following version :
Dev mode.......... spa
Quasar CLI........ v0.17.23
Quasar Framework.. v0.17.19

And the fiddle to reproduice it (do a research with the letter a is enough)
https://jsfiddle.net/smolinari/04feL195

Thank you for all

**Comment by rstoenescu** [comment]
Hi,

This has been taken care of in the QSelect (which has the autocomplete feature embedded) in v1. Closing the ticket.

**Comment by Lou-Der** [comment]
Hi Razvan,

Sorry to pull you back to the old version but is there a workaround in v0.17?   

I just noticed this issue and am not prepared to upgrade to v1 quite yet.....  

Thanks

**Comment by NRWhittle** [comment]
@Lou-Der I had a dirty workaround working for my project, but it most definitely has side-effects.

You can modifythe  `hide` method in `QAutocomplete` to be:

```
    hide () {
      if (!this.isWorking()) {
        this.results = []
        return this.$refs.popover.hide()
      }
      return Promise.resolve()
    }
```

The side effect here is that you no longer close the popover when the user selects something in the popover. So the solution for that is in your handler for `q-autocomplete`'s `@selected`:

`this.$refs.myAutocomplete.$refs.popover.hide();`

Like I said, it's dirty. But since dev for v1 is full steam ahead it might be all we can get.

**Comment by Lou-Der** [comment]
@NRWhittle Thanks for this.  It helped me understand where the problem was coming from.   

It seems `QAutocomplete` result list gets blurred when the scrollbars are touched as they are not part of the control.   If you remove lines 231 and 244 from `QAutocomplete.js` it will rely on the `popover` blur events to hide the results.

@rstoenescu would this be an acceptable fix to submit to v0.17?

Thanks!
